src/components/contributors-list.js

```diff
@@ -3,18 +3,18 @@ import { FormattedMessage } from 'react-intl';
 
 import { KeyValue } from '@folio/stripes/components';
 
-export default function ContributorsList({
+const ContributorsList = ({
   data,
   showInline,
   contributorsInlineLimit,
-}) {
+}) => {
   const contributorsByType = data.reduce((byType, contributor) => {
     byType[contributor.type] = byType[contributor.type] || [];
     byType[contributor.type].push(contributor.contributor);
     return byType;
   }, {});
   const showKeyValueList = () => (
-    <div>
+    <div data-testid="contributors-list-key-value">
```

Comment:
I'm not sure that you need this `div`. It will be better to remove this div, leave only fragment and test list items. (For mock of `contributorsByType` you can use array with one object.